third_party/terraform/resources/resource_container_node_pool.go.erb

```diff
@@ -106,12 +106,34 @@ var schemaNodePool = map[string]*schema.Schema{
 	},
 
 <% unless version == 'ga' -%>
-			"node_locations": {
-				Type:     schema.TypeSet,
-				Optional: true,
-				Computed: true,
-				Elem:     &schema.Schema{Type: schema.TypeString},
+	"node_locations": {
+		Type:     schema.TypeSet,
+		Optional: true,
+		Computed: true,
+		Elem:     &schema.Schema{Type: schema.TypeString},
+	},
+
+	"upgrade_settings": {
+		Type:     schema.TypeList,
+		Optional: true,
+		Computed: true,
```

Comment:
Can you explain why you made this `Computed`? If I don't send UpgradeSettings at all, I don't see it returned in the API, and Computed has the disadvantage of meaning that if a user removes this block from their config, it won't show a diff.

Reply:
You are right. I thought having `Computed` allowed the values to be interpolate elsewhere. I have removed it.